outputs.tf

```diff
@@ -48,7 +48,7 @@ output "cluster_iam_role_arn" {
 
 output "cluster_oidc_issuer_url" {
   description = "The URL on the EKS cluster OIDC Issuer"
-  value       = flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, [""]))[0]
+  value       = element(flatten(concat(aws_eks_cluster.this[*].identity[*].oidc.0.issuer, list(""))), 0)
```

Comment:
This change (+ [`kubeconfig_filename`](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R71) and [`oidc_provider_arn`](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061/files#diff-de6c47c2496bd028a84d55ab12d8a4f90174ebfb6544b8b5c7b07a7ee4f27ec7R76)) sounds not related to your bug. Can you please revert those changes.

Reply:
Fair enough, I added them so all outputs are consistent. I reverted this now.